rslib/src/decks/mod.rs

```diff
@@ -440,22 +440,23 @@ impl Collection {
         self.storage.get_deck_id(&machine_name)
     }
 
-    pub fn remove_deck_and_child_decks(&mut self, did: DeckID) -> Result<()> {
+    pub fn remove_decks_and_child_decks(&mut self, dids: Vec<DeckID>) -> Result<()> {
```

Comment:
Grr, GitHub ate my comment for some reason.

This would be better as `dids: &[DeckID]`, and passing the vec in as a reference. Then code like the following further down:

```rust
        col.remove_decks_and_child_decks(vec![top.id])?;
```

can be written more easily as

```
        col.remove_decks_and_child_decks(&[top.id])?;
```